ext4/xattr.py

```diff
@@ -117,6 +129,7 @@ class ExtendedAttributeEntry(ExtendedAttributeBase):
         "system.posix_acl_access",
         "system.posix_acl_default",
         "trusted.",
+        "",
```

Comment:
Could you point me at the ext4 docs that would explain why this change is needed?

Reply:
Interesting, Looking at the source it appears to be for `lustre`, whatever that is.

https://github.com/torvalds/linux/blob/cca7a0aae8958c9b1cd14116cb8b2f22ace2205e/fs/ext2/xattr.h#L20-L25